### DISCUSSION TITLE
What questions do you have about VDD?

### DISCUSSION THREAD

**Discussion by loub-lu** [T4RJ/community#3] [Values Driven Development]
What questions do you have about Values Driven Development? What might a Values Driven Development code review/dev process look like?

**Comment by holmesworcester** [comment]
One thing I think might be helpful to add is something about moving in solidarity with other groups in the human rights space to demand that existing commercial platforms (especially major platforms like messaging apps and social networks) prioritize security and privacy features like end-to-end encryption, timed deletion, and resilience against the device hacking tools used by governments. 

This seems important because groups in the RJ space use a lot of off-the-shelf commercial tools, and because developers are often really important voices on these questions, since they're who users and policymakers look to for technical expertise.

**Reply by BintLopez** [reply]
Yes! I love adding in the aspect of what features to look for in the tech we use and then to use this as a way to demands for those features in the tech we use.

**Comment by colinxfleming** [comment]
couple gut reax from the first read:
* jargon giraffe: I don't know what a PSO is lol. Choosing to believe it means `Party Stunt Operations`
* re `RJ critique sessions? RJ code review` I do think that one of the best things I've learned from our project has been to consider rj as an aspect of technical reviews; I think I've also chased some volunteers off by coming on a little too hard with this too early. (It might feel very weird making your first contribution to an open source codebase and then feeling like you're getting chastized for non-gender inclusive language for ex, to use a daria example where I missed the mark by accident from awhile ago.) So I think on balance this is good but needs to be very cautious, deliberate, and maybe delayed until people are more settled on a project. 
* That said I DO think an RJ review is much more valuable for project leadership/people structuring the work itself, who have more power and responsibility. lots of opportunity here.

**Reply by BintLopez** [reply]
Re your second point -- I've also had an issue with this. I think mine stemmed from unclear expectations in terms of onboarding. And thanks for pointing out the `party stunt operations` jargon lol.
